spec/models/bridge_spec.rb

```diff
@@ -1,12 +1,10 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
-require './spec/support/users_helper'
-require './spec/support/bridges_helper'
+require './spec/support/main_helper'
 
 RSpec.configure do |c|
-  c.include UsersHelper
-  c.include BridgesHelper
+  c.include MainHelper
```

Comment:
I won't make you change this but typically, you define a UserHelper, a BridgeHelper and then you mix them into a module called Helpers. 

So you include "Helpers" or you include one of the smaller modules if you just need that. This should be fine for us though.